protos/telemetry/telemetry.proto

```diff
@@ -165,3 +171,20 @@ message RcStatus {
     bool is_available = 2;
     float signal_strength_percent = 3;
 }
+
+message MavMessage {
+    enum Type {
+        EMERGENCY = 0;
+        ALERT = 1;
+        CRITICAL = 2;
+        ERROR = 3;
+        WARNING = 4;
+        NOTICE = 5;
+        INFO = 6;
+        DEBUS = 7;
+        UNKNOWN = 8;
```

Comment:
The thing is that PX4 only ever uses `INFO`, `WARNING`, and `CRITICAL`, so we might as well drop the others.

Reply:
Good to know. And we can bring the others back if needed later.

Reply:
Ah, as Jonas said, good to know. I'll drop them.